problem/problem_info.py

```diff
@@ -0,0 +1,16 @@
+import os.path
+from utils import config_info
+from problem.models import Problem, Testcase
+
+def get_problem(problem):
+    problem.testcase = Testcase.objects.filter(problem=problem)
+    return problem
+
+def has_special_judge_code(problem):
+    speJudge_path = config_info.get_config('path', 'special_judge_path')
```

Comment:
Store the config to a constant. Because each time we query config will involve filesystem read, which lead to some additional cost.